api/RequestTypes.d.ts

```diff
@@ -1311,7 +1311,7 @@ export interface Termvectors<T = RequestBody> extends Generic {
   body?: T;
 }
 
-export interface Update<T = RequestBody> extends Generic {
+interface UpdateRequest<T = RequestBody> extends GenericRequest {
```

Comment:
interestingly, we use `RequestBody` as a default value here, but `Record<string, any>` in `index.d.ts`.